### ISSUE TITLE
zookeeper start failed 

### ISSUE THREAD

**Issue by qifcheng** [google/gvisor#41]
I start the zookeeper container with the dockerhub official image, but it turns out repeated again and again. 
root@node23:/test# docker ps
CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS                            PORTS               NAMES
5348d914acd1        zookeeper           "/docker-entrypoint.…"   2 minutes ago       Restarting (127) 19 seconds ago                       some-zookeeper

while docker logs shows that:
root@node23:/test# docker logs 5348d914acd1
ZooKeeper JMX enabled by default
Using config: /conf/zoo.cfg
Error loading shared library libjli.so: No such file or directory (needed by /usr/lib/jvm/java-1.8-openjdk/jre/bin/java)
Error relocating /usr/lib/jvm/java-1.8-openjdk/jre/bin/java: JLI_Launch: symbol not found

docker version: 17.12.1-ce
OS version: Ubuntu 16.04
runtime: runsc

Can someone give me a hand? thanks.

**Comment by fvoznika** [comment]
The loader is not following RPATHs when running under gVisor for some reason. You can workaround this for now by instructing the loader to look under the Java's directory. Create the following file and map it to `/etc/ld-musl-x86_64.path` inside the container using `-v` option:
```
/lib
/usr/lib
/usr/local/lib
/usr/lib/jvm/default-jvm/jre/lib/amd64
/usr/lib/jvm/default-jvm/jre/lib/amd64/jli
/usr/lib/jvm/default-jvm/jre/lib/amd64/server
```
Then run:
`docker run -d --runtime=runsc -v /tmp/ld-musl-x86_64.path:/etc/ld-musl-x86_64.path zookeeper`

**Comment by amscanne** [comment]
This should have been fixed by c7d7312d6d86843d3c9b68abc2299ffb5d4840ba.
